### PR TITLE
Create pip install-able pkg

### PR DESCRIPTION
**Pull request by mfe7**
Thanks for releasing this great codebase! I moved the main python scripts into a `crown_ibp` directory to make them part of a python package, which allows other packages to easily import these useful methods/classes. You can now install this package with pip (because of the new `setup.py`) and thus run the scripts from other directories without having to change path variables. I also updated the README to reflect that you can call the train and eval scripts from other places after installing the package, but the config file paths still need to be dealt with.

No need to accept this pull request if you have different preferences for organizing the repo -- I can keep this fork open.

### CODE REVIEW

**Comment by huanzhang12** [comment]
@mfe7 Thank you for this great contribution! This is a very nice change that I always want to do but did not get a chance to make it. I will review your pull request and merge it.

BTW we have released a [new paper](https://arxiv.org/pdf/2002.12920)  which generalizes CROWN to a general computational graph algorithm. Our [new implementation](https://github.com/KaidiXu/auto_LiRPA), the `auto_LiRPA` library, allows you to compute bounds for any computational graph defined by PyTorch without any manual derivations, just like how easily you can obtain gradients in PyTorch. It should be much easier to use for any new projects. :slightly_smiling_face:

**Comment by mfe7** [comment]
Cool! I'll take a look! Thanks for sharing
